f2/src/PrivacyConsentPage.js

```diff
@@ -65,7 +65,7 @@ export const PrivacyConsentPage = () => {
                       type: 'saveFormData',
                       data: { consent: data },
                     })
-                    history.push('/howdiditstart')
+                    history.push('/anonymous')
```

Comment:
Right now we shouldn’t hook up thIs page since the rest of the anonymous flow isn’t there (for example, we say on this page that we won’t ask them for their contact details, but then we will later on). We can hook it up once the other parts are there.

Reply:
fixed

Reply:
fixed